src/linter/root.go

```diff
@@ -1134,4 +1191,16 @@ func (d *RootWalker) LeaveNode(n walker.Walkable) {
 	for _, c := range d.custom {
 		c.AfterLeaveNode(n)
 	}
+
+	if n == d.rootNode {
+		d.leaveFile()
+	}
+}
+
+// namespaceKey returns ns with several transformations applied.
+func namespaceKey(ns string) string {
+	key := strings.Replace(ns, "\\", "/", -1)
```

Comment:
In Windows path separator is actually "\\", to I suggest you use `os.PathSeparator` here if you use this key to match the file path.